### DISCUSSION TITLE
[Request feature] Add setting menu for Disable/Enable debug message on developer tool

### DISCUSSION THREAD

**Discussion by mean-cj** [Dhruv-Techapps/acf-docs#353] [General]
Hi , Thank for nice extension.

[Request feature] Please add setting menu for  "Disable/Enable"  show the debug message on developer tool 
It's disruptive for website development.

Thanks.

![image](https://github.com/Dhruv-Techapps/auto-clicker-auto-fill/assets/1191385/adef1d88-4f23-478b-bda7-dc00237ffcd0)

![image](https://github.com/Dhruv-Techapps/auto-clicker-auto-fill/assets/1191385/ab65d6e6-56eb-4e73-8585-071617131108)

**Comment by dharmesh-hemaram** [comment]
Sure we will add this feature after Feb release to stable. Currently we are focusing on fixing issuw which are there in DEV and later deploy to stable version

**Comment by dharmesh-hemaram** [comment]
# Filter out logs from Auto Clicker

## Steps
- open console settings
![image](https://github.com/Dhruv-Techapps/auto-clicker-auto-fill/assets/2290202/4cdc251b-8f81-46c3-be4e-fc1d14052fe2)
- Enabe `Selected Context only`
![image](https://github.com/Dhruv-Techapps/auto-clicker-auto-fill/assets/2290202/73dbf773-4582-46ad-a9d8-b9f2abae7d3a)
- Change the content of log from where it appear
![image](https://github.com/Dhruv-Techapps/auto-clicker-auto-fill/assets/2290202/bac1a90e-e041-4a19-8f6a-51e58c44d640)
- Select Auto Clicker - Auto Fill ( If you want to view logs only from Auto Clicker)
![image](https://github.com/Dhruv-Techapps/auto-clicker-auto-fill/assets/2290202/8ecf90dd-4d86-4777-8b9a-78549365b4ec)

**Comment by mean-cj** [comment]
Hi Dharmesh-Hemaram,

Thank you for your advice.

I understand how to filter console.log messages. 
Therefore, if it's possible disabling all debug messages from your plugin on plugin settings would be preferable.

CJ.

**Reply by dharmesh-hemaram** [reply]
I got it . I will check on how to solve this . Without adding additional condition

**Comment by dharmesh-hemaram** [comment]
Currently, we are not adding a condition to enable or disable the logging of low-level events. This decision is due to the high number of log events generated during execution. Introducing conditional checks before logging could increase the computational overhead, potentially slowing down the extension's performance.
